mobile/src/features/SetupWallet/useCases/RestoreWallet/RestoreWalletScreen.tsx

```diff
@@ -295,28 +265,20 @@ const WordSuggestionList = ({
   index: number
   onSelect: (index: number, word: string) => void
 }) => {
-  const {palette: p} = useTheme()
-  const {height: screenHeight} = useWindowDimensions()
-
-  const paddingBottom = React.useMemo(() => {
-    if (Platform.OS === 'android') {
-      return screenHeight < 700 ? 24 : 32
-    }
-    return screenHeight < 700 ? screenHeight * 0.01 : screenHeight * 0.05
-  }, [screenHeight])
+  const {palette: p, atoms: ta} = useTheme()
 
   return (
     <View
       style={[
+        ta.bg_color_max,
+        a.border_t,
         {
-          backgroundColor: p.bg_color_max,
           borderColor: p.gray_200,
-          borderTopWidth: 1,
-          paddingBottom: paddingBottom,
+          ...android(a.pb_sm),
```

Comment:
probably fixed with the new KBAV, should be tested

Reply:
@stackchain I had to add this to the KBAV. Android was still having an reside vertical offset even after the kb was gone. Tried to control it with isKeyboardOpen. Please see if it is okay. I have tested the changes. Look good.

keyboardVerticalOffset={
        isKeyboardOpen
          ? keyboardVerticalOffset != null
            ? keyboardVerticalOffset
            : defaultKeyboardOffset
          : 0
      }